webapp/channels/src/components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -997,7 +997,8 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
             draft.fileInfos = sortFileInfos(draft.fileInfos.concat(fileInfos), this.props.locale);
         }
 
-        this.handleDraftChange(draft, true);
+        this.props.actions.setDraft(StoragePrefixes.DRAFT + channelId, draft, channelId);
```

Comment:
I think we might want to rework how `handleDraftChange` here works. What we could do is supply the channelId to it so that we still clear the `saveDraftFrame`, and we don't have to repeat the code that's in `handleDraftChange` here.

Reply:
I did the changes. Let me know if this looks good for you : )